src/thor/unidirectional_astar.cc

```diff
@@ -267,27 +267,28 @@ inline bool UnidirectionalAStar<expansion_direction, FORWARD>::ExpandInner(
       best_path.second = cost.cost;
     }
 
+    // not_thru is the same for both trees
+    bool not_thru_pruning = pred.not_thru_pruning() || !meta.edge->not_thru();
```

Comment:
in fact, this algorithm never used not_thru pruning.. it recorded it in the label, but didn't make use of it.